src/System.Private.ServiceModel/tests/Scenarios/SelfHostWcfService/Program.cs

```diff
@@ -10,12 +10,18 @@ internal class Program
     {
         private static void Main(string[] args)
         {
-            using (ServiceHost host = new ServiceHost(
-                typeof(WcfService)))
+            using (ServiceHost wcfservicehost = new ServiceHost(
+              typeof(WcfService)))
             {
-                host.Open();
-                Console.WriteLine("Service is ready!");
-                Console.ReadKey();
+                wcfservicehost.Open();
+                Console.WriteLine("WcfService is ready");
+
+                using (ServiceHost userNameService = new ServiceHost(typeof(WcfUserNameService)))
```

Comment:
From a style point of view, I prefer having all the using statements together. Otherwise adding more services will cause a deep nested mess. E.g.
```csharp
using (ServiceHost wcfservicehost = new ServiceHost(typeof(WcfService))) 
using (ServiceHost userNameService = new ServiceHost(typeof(WcfUserNameService)))
{
    wcfservicehost.Open();  
    Console.WriteLine("WcfService is ready");
    userNameService.Open();
    Console.WriteLine("Wcf user name service is ready");
    Console.WriteLine( "Press ENTER to close services...")
    Console.ReadLine();
}
```
This way you only indent the one time, and everything looks cleaner and at least for me is easier to follow.